tests.js

```diff
@@ -124,6 +124,10 @@ var basicTests = function() {
                     [];
                 }),
             "Empty array matches [,]");
+            
+        ok(Structured.match("var myVar = \"abc I love strings!\"", function() {
+            var _ = "abc I love strings!";
+        }), "\"text\" will match 'text'")
```

Comment:
It looks like this is checking `"` against `"`.  It would be good to some additional tests that check `"` against `'` and vice versa.